test/apiv2/python/rest_api/test_v2_0_0_container.py

```diff
@@ -458,6 +458,55 @@ def test_host_config_port_bindings(self):
         self.assertTrue("8080/tcp" in inspect_response["HostConfig"]["PortBindings"])
         self.assertFalse("8081/tcp" in inspect_response["HostConfig"]["PortBindings"])
 
+    def test_host_config_cgroupns_mode(self):
+        # Test private mode
+        r = requests.post(
```

Comment:
instead of duplicating the same requests twice I guess a for loop would help
```
for mode in ["private", "host"]:
```

Reply:
thanks